modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/servlet/filter/DLVideoEmbedFilter.java

```diff
@@ -139,12 +138,15 @@ private FileEntry _getFileEntry(
 		throws PortalException {
 
 		if (_PATH_SEPARATOR_FILE_ENTRY.equals(pathParts.get(1))) {
-			Optional<FileEntry> fileEntryOptional = _resolveFileEntry(
+			FileEntry fileEntry = _resolveFileEntry(
```

Comment:
@JanisZhang I think this commit should belong to (be merged into) the last commit?